site/static/examples/kind-with-registry.sh

```diff
@@ -23,7 +23,11 @@ EOF
 
 # connect the registry to the cluster network
 # (the network may already be connected)
-docker network connect "kind" "${reg_name}" || true
+networks=$(docker inspect --format='{{.NetworkSettings.Networks}}' "$reg_name")
+kindnetwork=$(echo "${networks}" | grep 'kind:')
+if [ "${networks}" != "${kindnetwork}" ]; then
+    docker network connect "kind" "${reg_name}" || true
+fi
```

Comment:
i think it is possible to do something like:

```
kindnetwork=$(docker inspect --format='{{json .NetworkSettings.Networks.kind}}' "$reg_name")
if [ "${kindnetwork}" != "null" ]; then
    docker network connect "kind" "${reg_name}" || true
fi
```

(untested)

Reply:
I don't think you can use a variable network name in the `docker inspect --format` specifier. I was not able to find a way to make that work.

Reply:
hm, this article here references a network like that:
https://www.baeldung.com/ops/docker-network-information

`{{.NetworkSettings.Networks.network1....`

Reply:
Hmm, I tried it again and can only assume I typo'd something when I was trying to get it before. This works, and returns the string "null" if `.NetworkSettings.Networks.kind` can't be found, otherwise all the details of the network. So this would be a much simpler check.

I'll get this updated to use that approach instead. Thanks!